tests/containers/privileged_mode.pm

```diff
@@ -31,7 +31,11 @@ sub run {
 
     record_info('Test', 'Launch a container with privileged mode');
 
-    # /dev is only accessible in privileged mode
+    my $devices = script_run("$runtime run --rm --privileged $image ls /dev");
+    record_info("Devices (privileged)", $devices);
+    $devices = script_run("$runtime run --rm $image ls /dev");
```

Comment:
LGTM
I'd suggest to use a different variable for unpriviledged, for future possible uses.

Reply:
> LGTM I'd suggest to use a different variable for unpriviledged, for future possible uses.

This is just for data collection.  I will revert this MR after finding a device that we could use across all architectures instead of /dev/bus